src/python/tests/test_orthology_benchmark.py

```diff
@@ -0,0 +1,234 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `orthology_benchmark.py` script.
+
+Typical usage example::
+
+    $ pytest test_orthology_benchmark.py
+
+"""
+
+from contextlib import nullcontext as does_not_raise
+from importlib.abc import Loader
+from importlib.machinery import ModuleSpec
+from importlib.util import module_from_spec, spec_from_file_location
+import os
+from pathlib import Path
+import sys
+from typing import ContextManager, Dict, List
```

Comment:
You'd need to import `Optional` here if you decide to use it later in the script.
```suggestion
from typing import ContextManager, Dict, List, Optional
```